### ISSUE TITLE
Chat -> Action -> Add/remove user from "highlight user"

### ISSUE THREAD

**Issue by gr3ger** [FrankerFaceZ/FrankerFaceZ#907]
**Is your feature request related to a problem? Please describe.**
As a mod you want to keep track of some people, so being able to add this action to user context/inline or wherever you want it would be a very nice thing to have.

**Describe the solution you'd like**
A new action added to Chat -> Actions -> In-Line/UserContext that will add/remove it to "user highlight" on the same profile the action is a part of.

**Describe alternatives you've considered**
The alternative is to copy/paste the name manually into Chat -> Filtering

**Comment by GabrielDuarteM** [comment]
yeah, that would be amazing! it's funny cause I was actually about to open this exact feature request ahahaha

For now, I've been using the viewer card highlight (Chat >> Viewer Cards >> Highlight messages from users with open viewer cards), which gives me the ability to track one user if I keed their user card open somewhere, but sometimes I want to keep track of more then one, and because I can't open more then one viewer card, this feature would be really handy...

**Comment by gr3ger** [comment]
That sadly doesn't work in mod view though :/
I've been trying to implement this myself but my JS is rusty at best.
